exchanges/persisted/src/persistedExchange.ts

```diff
@@ -118,7 +123,8 @@ export const persistedExchange =
   ({ forward }) => {
     if (!options) options = {};
 
-    const preferGetForPersistedQueries = !!options.preferGetForPersistedQueries;
+    const preferGetForPersistedQueries =
+      options.preferGetForPersistedQueries ?? true;
```

Comment:
Two things 🤔 I think we shouldn't change the default here. I believe I simply typo'd the default value in the TSDocs since I wrote many of them all at once 😅

The other thing is that we currently don't use the bullish coalescing operator or chaining operators. Currently, due to some JS engine edge cases transpilation for them is still turned on, and that leads to some unnecessary code in the bundle. Either way, I suppose, since the default was wrong this could be removed ✌️


```suggestion
      options.preferGetForPersistedQueries;
```

Reply:
Thanks! That makes sense. Fixed in https://github.com/urql-graphql/urql/pull/3192/commits/a9d74916122a4056d7faf27ea3e3fd1b6160ed41.

I also added the `&& preferGetForPersistedQueries` check back to the `persistedOperation.context.preferGetMethod` assignment:
1. So that we don't set `persistedOperation.context.preferGetMethod` to `undefined`
2. To continue treating a falsey value as the default